src/pages/AddTransactionPage.ts

```diff
@@ -0,0 +1,114 @@
+import IPage from './IPage';
+import MenuPage from './MenuPage';
+import { IPrompter } from '../Prompter';
+import { IUserIO } from '../CLI';
+import { Answers } from 'inquirer';
+import { userTable, transactionTable } from '../tables';
+import { IUser } from '../tables/UserTable';
+import chalk from 'chalk';
+
+export default class AddTransactionPage implements IPage {
```

Comment:
Think about what this page does, and if something violates the single responsibility principle